ratbagd/ratbagd.c

```diff
@@ -520,10 +524,12 @@ int main(int argc, char *argv[])
 	int r;
 
 	if (argc > 1) {
-		if (streq(argv[1], "--verbose")) {
+		if (streq(argv[1], "--verbose=raw")) {
+			verbose = true;
```

Comment:
Shouldn't this be `verbose_raw = true`?

Reply:
doh. yes, thanks. pushed the fix directly to master